app/controllers/team_repositories_controller.rb

```diff
@@ -33,6 +33,21 @@ def destroy
     end
   end
 
+  # POST :team_id/repositories/:repository_id/multiple_update
+  def multiple_update
+    service_call = Repositories::MultipleShareUpdateService.call(repository_id: @repository.id,
+                                                                 user_id: -1,
```

Comment:
Why is this -1, wouldn't this resolve into `nil` in service?

Reply:
Wooow 😲... thanks.